integrations/logging/datadog/README.md

```diff
@@ -0,0 +1,49 @@
+## Datadog
+
+1. Create a free 14-days Datadog account at [datadoghq.com](https://www.datadoghq.com/free-datadog-trial/#)
```

Comment:
What happens after 14 days? We just lose access?

Reply:
If you go with the free account yes

Reply:
Hm... I don't like that it does not have a free time-unlimited mode for testing :(

Reply:
If we want to go that route, we could request a [Datadog sandbox account](https://docs.datadoghq.com/developers/integrations/api_integration/#build-your-integration) for development, but if I understand correctly, that means being part of the [Datadog Partner Network](https://partners.datadoghq.com/s/login/?ec=302&startURL=%2Fs%2F)

Maybe it's something we want to investigate in the future, but I wouldn't do it right now.